tooling/operation.js

```diff
@@ -265,6 +266,24 @@ class Operation {
     return getParametersAsJsonSchema(this.path, this.schema, this.oas, globalDefaults);
   }
 
+  /**
+   * Get a single response for this status code, formatted as JSON schema
+   * @param {*} statusCode
+   * @returns
+   */
+  getResponseAsJsonSchema(statusCode) {
+    return getResponseAsJsonSchema(this, this.oas, statusCode);
+  }
+
+  /**
+   * Get an array of all valid response status codes for this operation
+   * @param {*} statusCode
+   * @returns
+   */
+  getResponseStatusCodes() {
+    return Object.keys(this.schema.responses);
```

Comment:
Just in case there's no `responses`:

```suggestion
    return this.schema.responses ? Object.keys(this.schema.responses) : [];
```